docs/Installation-Windows.md

```diff
@@ -80,92 +80,92 @@ Make sure you are connected to the internet and then type in the Anaconda Prompt
 pip install .
 ```
 
-This will complete the installation of all the required Python packages to run ML-Agents.  
+This will complete the installation of all the required Python packages to run ML-Agents. 
 
-## (Optional) GPU Training using ML-Agents 
+## (Optional) Step 4: GPU Training using ML-Agents 
 
-Not required to use v0.3 for ML-Agents.  This is a guide for advanced users who want to train using GPUs.  Additionally, you will need to check if your GPU is CUDA compatible.  Please check Nvidia's page [here](https://developer.nvidia.com/cuda-gpus).
+Not required to use v0.3 for ML-Agents. This is a guide for advanced users who want to train using GPUs. Additionally, you will need to check if your GPU is CUDA compatible. Please check Nvidia's page [here](https://developer.nvidia.com/cuda-gpus).
 
 As of ML-Agents v0.3, only CUDA 8 and cuDNN 6 is supported.
 
-## (Optional) Step 1: Install Nvidia CUDA toolkit
+### Install Nvidia CUDA toolkit
```

Comment:
Would it make sense to add some indexing ? 
```
### A. Install Nvidia CUDA toolkit
```
or 
```
### 1) Install Nvidia CUDA toolkit
```

Reply:
Since its an optional section, I felt like any sub-indexing would be confusing.